Deployment/Create-RunSettingsFile.ps1

```diff
@@ -2,26 +2,26 @@
     .SYNOPSIS
     Retrieves necessary values from Azure and creates .runsettings-file
 
-    .PARAMETER ResourceGroup
-    Name of the resource group that has the web app deployed
+    .DESCRIPTION
+    The Azure environment should already exist and the webapp already deployed.
 
-    .PARAMETER WebAppName
-    Name of the target web app. If not set, resource group name is used.
+    .PARAMETER SettinsFile
+    Settings file that contains resource group name, web app name etc. Defaults to 'developer-settings.json'
```

Comment:
We can't use `SettingsFile` in `Create-RunSettingsFile.ps1`  because it's used in `Jenkinsfile`. I would rather use raw WebAppName to avoid creating a `developer-settings.json` in pipeline.

Reply:
oh yes, sure